### ISSUE TITLE
Expand support for stage stageAlign and scaleMode

### ISSUE THREAD

**Issue by korinVR** [openfl/openfl#136]
OpenFL flash.display.StageAlign seems to have two problems.

(1) StageAlign does not work on Window target.
I remember Android (and iOS?) target also has this issue.

(2) Flash StageAlign has an undocumented default parameter "" but OpenFL can't use this.
(ref. http://gamedev.stackexchange.com/questions/37559/how-to-center-a-game-in-flash-projector-like-adamatomics-capsule)

Here is a test code.

```
<window width="640" height="480" if="flash" />
<window width="640" height="480" fullscreen="true" if="desktop" />

class Main extends Sprite
{
    public function new()
    {
        super();

        graphics.beginFill(0x00FF00);
        graphics.drawRect(0, 0, 640, 480);
        graphics.endFill();

        stage.scaleMode = StageScaleMode.SHOW_ALL;

        // Default stage alignment is 
        //     "TL" (OpenFL) - top left
        //     "" (Flash/ActionScript) - center
        trace(stage.align);

        // This does not work on Windows target
//      stage.align = StageAlign.RIGHT;

        // Cause a compile error
//      stage.align = "";

        // Works on Flash target (stage will be centered)
//      untyped stage.align = "";
    }
}
```

**Comment by jgranick** [comment]
Redirecting to https://trello.com/c/BtbyXfw9/13-add-support-for-more-stage-stagealign-scale-modes

**Comment by joshtynjala** [comment]
Closing. scaleMode was implemented in 098b1583d8a6d4a776de3963c9c6c5c5bf8b57c7.
